### DISCUSSION TITLE
V10 support?

### DISCUSSION THREAD

**Discussion by geoidesic** [moo-man/WFRP4e-FoundryVTT#1478] [Q&A]
The system I have installed currently gives a compatibility warning for foundry v10. What's up with that?

**Comment by moo-man** [comment]
Not sure, the system and modules are all compatible. To dig in deeper I'd need to know more details, like error messages

**Reply by geoidesic** [reply]
Never mind. I got this wrong. Not sure why you closed my other question tho, which is unresolved. You can close this one but I would appreciate it if you re-opened the other one about the missing journals.

**Reply by moo-man** [reply]
Sorry, actually thought this was an issue, not a discussion
